docs/services/ml/_index.md

```diff
@@ -24,6 +24,12 @@ For some models, like the [Triton MLModel](/modular-resources/examples/triton/)
 | ----- | ----------- |
 | [`"tflite_cpu"` model](#create-an-ml-model-service) | Runs a tensorflow lite model that you have [trained](/manage/ml/train-model/) or [uploaded](/manage/ml/upload-model/). |
 
+## Required Components
+
+{{< cards >}}
+{{< relatedcard link="/components/board/" >}}
+{{< /cards >}}
```

Comment:
I don't think this is true. I think there are no required components. The board does not need to be configured as a component.  So I think :
```suggestion
None.
```